stable/mongodb-replicaset/values.yaml

```diff
@@ -27,6 +27,20 @@ image:
   tag: 3.6
   pullPolicy: IfNotPresent
 
+# Specs for busybox image
+
+initContainers:
+  busybox:
+    repository: busybox
```

Comment:
While you're at it, please add a tag with a stable version as well. Also, follow naming best practices. I'd suggest this:

```yaml
copyConfigImage:
  repository:
  tag:
  pullPolicy:
```

Reply:
Fixed